docs/src/main/asciidoc/datasource.adoc

```diff
@@ -296,16 +296,15 @@ Because of this, you can also use `javax.sql.DataSource` as the injected type.
 
 ===== Oracle considerations
 
-As documented in https://github.com/quarkusio/quarkus/issues/36265[issue #36265],
-Oracle has a very weird behavior of committing the uncommitted transactions on connection closing.
+As documented in link:https://github.com/quarkusio/quarkus/issues/36265[issue #36265],
+Oracle unexpectedly commits uncommitted transactions when closing a connection.
+This means that when stopping Quarkus, in-progress transactions might be committed even if they are incomplete.
 
-Which means that when stopping Quarkus for instance, in progress transactions might be committed even if incomplete.
+Since this behavior is unexpected and could lead to data loss, an interceptor rolls back any unfinished transactions when closing a connection, unless XA transactions are used, in which case the transaction manager handles the rollback.
 
-Given that is not the expected behavior and that it could lead to data loss, we added an interceptor that rolls back any unfinished transactions at connection close,
-provided you are not using XA (in which case the transaction manager handles things for you).
+If the behavior introduced in 3.18 causes issues for your workload, disable it by setting the `-Dquarkus-oracle-no-automatic-rollback-on-connection-close` system property to `true`.
+If possible, report your use case in our link:https://github.com/quarkusio/quarkus/issues[issue tracker] so we can adjust this behavior if needed.
```

Comment:
Two "ifs" in the same sentence?

At any rate I'd remove "if possible". I don't see why it wouldn't possible for some to report that they need this. And if they don't we'll eventually remove this flag, so they really should do it.

For example:

```suggestion
Make sure to report your use case in our link:https://github.com/quarkusio/quarkus/issues[issue tracker] so we can adjust this behavior if needed, for example with more permanent settings.
```